### PR TITLE
ENG-7016 - Add Long/Lat tracking in Metadata

### PR DESCRIPTION
**Pull request by kevinsites**

### CODE REVIEW

File: NeuroID/NeuroIDClass/NeuroID.swift [hunk 1]
```diff
@@ -94,6 +95,9 @@ public enum NeuroID {
         
         // Reset tab id on configure
         setUserDefaultKey(Constants.storageTabIDKey.rawValue, value: nil)
+
+
+        locationManager = LocationManager()
```
Comment (kishisaka-nid) [inline NeuroID/NeuroIDClass/NeuroID.swift:100]: Do we need to a getter for the locationManager so the client can authorize its use? ```  locationManager.requestWhenInUseAuthorization()```

**Review by priya-xavier — APPROVED** [review]

File: NeuroID/NeuroIDClass/NeuroID.swift (hunk 1 above)
Comment (priya-xavier) [reply NeuroID/NeuroIDClass/NeuroID.swift:100]: I wasn't asked permission when I tried this on my device and I only see the defaults of -1.0 for lat long in the logs. So it prob needs permission.

Comment (kishisaka-nid) [reply NeuroID/NeuroIDClass/NeuroID.swift:100]: Yeah I was seeing this too. It looks like you have to call requestWhenInUseAuthoriztion() else the self.Authoriztion status is not determined and the location manager is not started.

Comment (kevinsites) [reply NeuroID/NeuroIDClass/NeuroID.swift:100]: Correct, but we aren't concerned with asking for permission, we just want to see if permission is granted by the user and then collect. We assume our customer will be asking for the location permissions and we just want to collect based on allowed scopes. I can show you both a demo that gets location if you'd like

**Review by priya-xavier — APPROVED** [review]
LGTM!

**Review by kishisaka-nid — APPROVED** [review]

**Pull request merged**